src/extension.ts

```diff
@@ -895,7 +895,19 @@ export function findKindNameOrPrompt(resourceKinds: kuberesources.ResourceKind[]
 }
 
 function promptKindName(resourceKinds: kuberesources.ResourceKind[], descriptionVerb, opts, handler) {
-    vscode.window.showInputBox({ prompt: "What resource do you want to " + descriptionVerb + "?", placeHolder: 'Empty string to be prompted' }).then((resource) => {
+    let placeHolder: string = 'Empty string to be prompted';
+    let prompt: string = "What resource do you want to " + descriptionVerb + "?";
+    if (opts) {
+        // see if the options has 'placeHolder' string, if so use it instead of default one
+        if (opts.placeHolder) {
```

Comment:
Style suggestion: consider `placeholder = opts.placeHolder || placeholder;` (and similarly for prompt).

Reply:
fantastic will do.